tests/consistent-package-json.test.ts

```diff
@@ -139,6 +139,18 @@ packages.forEach(
           it('specifies the expected types', () => {
             expect(packageJSON.types).toBe(expectedPackageJSON.types);
           });
+
+          it('specifies the expected exports', () => {
```

Comment:
It is possible for packages to have an exports key that does not contain `"."` - e.g. `graphql-persisted`,  so I don't think this test is correct.